manifests/klusterlet/managed/klusterlet-registration-clusterrolebinding-addon-management.yaml

```diff
@@ -0,0 +1,14 @@
+# ClusterRoleBinding for registration addon management permissions.
+# TODO: remove this ClusterRoleBinding, create RoleBindings and bind ClusterRole and ServiceAccount to user defined addon namespaces.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
```

Comment:
we should not need this when addon-management feature gate is not enabled.

Reply:
do you want me to add this to comments?

Reply:
@qiujian16 add more comments for that.